### DISCUSSION TITLE
Non wyze cameras

### DISCUSSION THREAD

**Discussion by dersgniw** [mrlt8/docker-wyze-bridge#969] [Q&A] [answered]
I'm wondering if there is a way to get other cameras to work with the bridge.  I was starting to setup Agent DVR, then I realized, MAYBE, the wyze bridge interface could display a couple of my foscam cameras.

Is this possible?

**Comment by mrlt8** [comment, marked as answer]
Not right now, but it could be possible. I do want to make the bridge more modular so we can add support for other wyze clones like roku, so we could potentially add support for a generic RTSP proxy at the same time.

**Reply by bshiers** [reply]
That sounds great.  I've been trying to get your container working with the Roku cameras (which are just Wyze cameras, but it looks to me like they use a different URL to authenticate and get data).  I'm still working through seeing if I can modify this to work with those.
